res/values-ga/strings.xml

```diff
@@ -257,7 +257,7 @@
     <string name="Server_Restart_">"Athshlánú Freastalaí:"</string>
     <string name="GOAL_">"SPRIOC!"</string>
     <string name="SOCCER">"SAOIRE"</string>
-    <string name="DRAW">"DRAW"</string>
+    <string name="DRAW">"Tarraing"</string>
```

Comment:
Please keep the 100% capital letters here before approval.

Reply:
It's not my fault if it's not solved, I've done what I have to do for the rest on you